code/modules/mob/living/silicon/ai/ai_mob.dm

```diff
@@ -1005,7 +1005,7 @@ GLOBAL_LIST_INIT(ai_verbs_default, list(
 	flat_icon.ColorTone(rgb(125,180,225)) // Let's make it bluish.
 	flat_icon.ChangeOpacity(0.5) // Make it half transparent.
 
-	if(A.Height() == 64)
+	if(flat_icon.Height() == 64)
```

Comment:
What issue this caused? Its only related to cyborgs or in general? ~~this is totally not me just doing a review to make the stale label go away~~

Reply:
In the initial commit for this PR, I passed in `robot.icon` directly, mistakenly believing that it was of the `icon` type. This resulted in a runtime error (`DEBUG: Runtime in code/modules/mob/living/silicon/al/ai_mob.dm:1008: Cannot execute 'icons/mob/robots.dmi'.Height().`) and invisible hologram whenever borgs tried to use holopads. This change, along with passing a false `safety` value to `getHologramIcon` when calling it for cyborgs, fixed the runtime error and allowed borg holograms to be visible, because `/proc/getHologramIcon` sets `flat_icon` to `new(A)` when `safety` is false.

A very ugly make-it-work solution. I've updated the PR, reverting this line change, and removing the false safety parameter and properly calling `icon(robot.icon)` when calling `getHologramIcon` instead.